### ISSUE TITLE
[dev] unpuppet/ooc command missing from default merged cmdset

### ISSUE THREAD

**Issue by strikaco** [evennia/evennia#1865]
#### Brief summary of issue / Description of requested feature:
In `MULTISESSION_MODE=2`, one can create accounts and characters and puppet them just fine, and even talk on the OOC channel while OOC, but the related commands for unpuppeting and using the OOC channel while in-character are lost during merging.

Account object, no puppet:
````
Stored Cmdset(s):
 commands.default_cmdsets.AccountCmdSet [DefaultAccount] (Union, prio -10)
Merged Cmdset(s):
 evennia.commands.cmdset.CmdSet [ChannelCmdSet] (Union, prio 101)
 evennia.commands.cmdset.CmdSet [DefaultAccount] (Union, prio -10)
 commands.default_cmdsets.SessionCmdSet [DefaultSession] (Union, prio -20)
Commands available to johnny (result of Merged CmdSets):
   addcom, admin, admin, allcom, cboot, ccreate, cdesc, cdestroy, cemit,
  channels, charcreate, chardelete, clock, color, cwho, delcom, examine,
  grapevine2chan, help, hints, hints, ic, irc2chan, ircstatus, look, mudinfo,
  mudinfo, news, news, nick, option, page, password, public, public, py,
  quell, quit, reload, reset, rss2chan, sessions, shutdown, style,
  userpassword, who
````

Character object, puppeted:
````
Stored Cmdset(s):
 commands.default_cmdsets.CharacterCmdSet [DefaultCharacter] (Union, prio 0)
Merged Cmdset(s):
 evennia.commands.cmdset.CmdSet [ChannelCmdSet] (Union, prio 101)
 commands.default_cmdsets.CharacterCmdSet [DefaultCharacter] (Union, prio 0)
 commands.default_cmdsets.AccountCmdSet [DefaultAccount] (Union, prio -10)
 commands.default_cmdsets.SessionCmdSet [DefaultSession] (Union, prio -20)
Commands available to johnny (result of Merged CmdSets):
   about, access, accounts, addcom, admin, alias, allcom, ban, batchcode,
  batchcommands, boot, cboot, ccreate, cdesc, cdestroy, cemit, channels,
  charcreate, chardelete, clock, cmdsets, color, copy, cpattr, create, cwho,
  delcom, desc, destroy, dig, drop, emit, examine, find, force, get, give,
  grapevine2chan, help, hints, home, ic, inventory, irc2chan, ircstatus, link,
  lock, look, mudinfo, mvattr, name, news, nick, objects, open, option, page,
  password, perm, pose, public, py, quell, quit, reload, reset, rss2chan, say,
  script, scripts, server, service, sessions, set, setdesc, sethelp, sethome,
  shutdown, spawn, style, tag, tel, tickers, time, tunnel, typeclass, unban,
  unlink, userpassword, wall, west, whisper, who, wipe
````

#### Error output / Expected result of feature
`Command 'ooc' is not available. Maybe you meant "ooc", "color" or "cboot"?`
`Command '@ooc' is not available. Maybe you meant "ooc", "color" or "cboot"?`

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
Clean install, dev branch (prob master too).

**Comment by strikaco** [comment]
On further investigation it appears this is a conflict between the `ooc` command and the alias for the `Public` channel. 

`ooc hi` *does* emit a message to the `Public` channel, but a plain `ooc` with no args yields a command-not-found error. Removing the alias from the channel makes the naked `ooc` work as intended.

**Comment by kovitikus** [comment]
I have the exact same issue. I puppeted a character and now I can't unpuppet it either through the unpuppet command or ooc. I also receive:
`Command 'ooc' is not available. Maybe you meant "ooc", "cboot" or "color"?`

Here's a listing of my help entries while on the puppeted character. [https://pastebin.com/P5zSFLZL](url)

I also have a fresh install.

**Comment by strikaco** [comment]
For a workaround, remove the 'ooc' alias from the `Public` channel:

`@py from typeclasses.channels import Channel;c=Channel.
objects.get(db_key="Public");c.aliases.remove('ooc')`

**Comment by Griatch** [comment]
I have removed the "ooc" alias from the default public channel. As mentioned, this is a name collision between the channel alias and the command, and the channel has the higher priority. With the closing of #1864, one can now simply `cdestroy public` and reload the server to rebuild the `public` channel anew. If having a game with existing subscribers, delete the channel's "ooc" alias manually. 
Closing, not much more to do about this one.
